cdap-validation/bin/test_helpers.py

```diff
@@ -0,0 +1,84 @@
+import os
+import base64
+import urllib2
+import json
+import sys
+
+def usage():
+    print "\nThis is the usage function\n"
+    print 'Usage: \n'+sys.argv[0]+' [-v, --verbose] -c, --cluster \'<cluster>\' [-m, --modules \'<modules>]\' -u --user <user:password> -U --uri <URI>\n'
+
+    print """
+    -h, --help  help menu
+    -c, --cluster       cluster name used in API calls to the install manager
+    -m, --modules       run modules, e.g. --modules 'module1;module2'
```

Comment:
I added a bit more info in the usage section.
I have a link to the design on the wiki at the top.
I added a basic README file for modules in the modules directory that shows basic structure"